config/webpack.config.dev.js

```diff
@@ -67,7 +67,6 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        include: srcPath,
```

Comment:
Let's explicitly include `srcPath` *and* related `node_modules` folder. This helps us know for sure where things may be located.